src/Libraries/RevitNodes/Filter/OverrideGraphicSettings.cs

```diff
@@ -50,18 +44,24 @@ internal OverrideGraphicSettings(Autodesk.Revit.DB.OverrideGraphicSettings inter
         /// <param name="projectionFillPattern">Projection fill pattern</param>
         /// <param name="cutLinePattern">Cut line pattern</param>
         /// <param name="projectionLinePattern">Projection line pattern</param>
-        /// <returns>OverrideGraphicSettings</returns>
+        /// <param name="transparency">Transparency as integer between 1-100.</param>
+        /// <param name="detailLevel">Detail Level setting, ex: Coarse, Fine etc.</param>
+        /// <param name="halftone">Halftone. True = halftone.</param>
+        /// <returns name="overrides">Override Graphic Settings</returns>
         public static OverrideGraphicSettings ByProperties(
             [DefaultArgumentAttribute("null")]DSCore.Color cutFillColor,
             [DefaultArgumentAttribute("null")]DSCore.Color projectionFillColor,
             [DefaultArgumentAttribute("null")]DSCore.Color cutLineColor,
             [DefaultArgumentAttribute("null")]DSCore.Color projectionLineColor,
-            [DefaultArgumentAttribute("null")]FillPatternElement cutFillPattern,
-            [DefaultArgumentAttribute("null")]FillPatternElement projectionFillPattern,
-            [DefaultArgumentAttribute("null")]LinePatternElement cutLinePattern,
-            [DefaultArgumentAttribute("null")]LinePatternElement projectionLinePattern,
+            [DefaultArgumentAttribute("null")]Revit.Elements.Element cutFillPattern,
```

Comment:
because unwrapper Revit API elements are not accessible in Dynamo...not for everyday users at least.